docs/src/pages/_app.page.tsx

```diff
@@ -86,11 +86,15 @@ function MyApp({ Component, pageProps }) {
 
       <div className={isHomepage ? `docs-home` : ''}>
         <ThemeProvider theme={baseTheme} colorMode={colorMode}>
-          <GlobalNav
-            rightLinks={RIGHT_NAV_LINKS as NavMenuItem[]}
-            leftLinks={LEFT_NAV_LINKS as NavMenuItem[]}
-            currentSite="UI Library"
-          />
+          {/* Remove this conditional post-launch */}
+          {process.env.NEXT_PUBLIC_SHOW_GLOBAL_NAV ? (
```

Comment:
Not super important but we could use the _featureFlags.ts_ file to contain the lookup logic: https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/utils/featureFlags.ts

Reply:
Didn't know about this! I added it to the feature flag. Have we tested the react native env variable? From my understanding the variable needs to start with NEXT_PUBLIC if you use it in the client.

Reply:
Super old now but here is the dev version: https://ui-react-native-add-react-native-docs-home.d2wnvax2ctai8z.amplifyapp.com/

Reply:
hmm weird... I tried removing NEXT_PUBLIC from my local env var and it stopped working...

Reply:
Ahhhh nevermind, I needed to add it to the next config AND set the env var